src/app/common/electrum/client.ts

```diff
@@ -0,0 +1,35 @@
+import ElectrumClient from 'electrum-client-sl';
+
+import type { DefaultNetworkConfigurations } from '@shared/constants';
+
+function getElectrumConfig(network: DefaultNetworkConfigurations) {
+  switch (network) {
+    case 'devnet':
+      return {
+        protocol: 'tcp',
+        host: 'localhost',
+        port: 50001,
+      };
+
+    case 'mainnet':
+      return {
+        protocol: 'ssl',
+        host: 'fortress.qtornado.com',
+        port: 443,
+      };
+
+    default:
+      return {
+        protocol: 'ssl',
+        host: 'blackie.c3-soft.com',
+        port: 57006,
+      };
+  }
```

Comment:
@kyranjamie just want to make sure it's fine to use these 3rd party services for Electrum.